service/src/main/java/tds/config/repositories/impl/ConfigRepositoryImpl.java

```diff
@@ -3,23 +3,41 @@
 import org.slf4j.Logger;
 import org.slf4j.LoggerFactory;
 import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.dao.DataAccessException;
 import org.springframework.dao.EmptyResultDataAccessException;
+import org.springframework.dao.IncorrectResultSizeDataAccessException;
 import org.springframework.jdbc.core.BeanPropertyRowMapper;
 import org.springframework.jdbc.core.namedparam.MapSqlParameterSource;
 import org.springframework.jdbc.core.namedparam.NamedParameterJdbcTemplate;
 import org.springframework.jdbc.core.namedparam.SqlParameterSource;
 import org.springframework.stereotype.Repository;
 import tds.config.ClientSystemFlag;
 import tds.config.ClientTestProperty;
+import tds.config.TimeLimits;
 import tds.config.repositories.ConfigRepository;
 
 import javax.sql.DataSource;
 import java.util.List;
+import java.util.Optional;
 
 @Repository
 public class ConfigRepositoryImpl implements ConfigRepository {
     private static final Logger LOG = LoggerFactory.getLogger(ConfigRepositoryImpl.class);
     private final NamedParameterJdbcTemplate jdbcTemplate;
+    private final String TIMELIMITS_BASE_SQL =
+            "SELECT \n" +
+            "   clientname, \n" +
+            "   _efk_testid AS assessmentId, \n" +
+            "   environment, \n" +
+            "   opprestart AS examRestartWindowMinutes, \n" +
+            "   oppdelay AS examDelayDays, \n" +
+            "   interfacetimeout AS interfaceTimeoutMinutes, \n" +
+            "   requestinterfacetimeout AS requestInterfaceTimeoutMinutes, \n" +
+            "   tacheckintime AS taCheckinTimeMinutes \n" +
+            "FROM \n" +
+            "   session.timelimits \n" +
+            "WHERE \n" +
```

Comment:
I don't like it either; removed `TIMELIMITS_BASE_SQL` and put the queries back in their respective methods.